partials/twitter.php

```diff
@@ -1,12 +1,24 @@
 <?php 
   $twitter_handle = IGV_get_option( '_igv_twitter_handle' ); 
 ?>
-<section class="row">
+<section id="twitter-feed" class="">
 
   <?php
     if ($twitter_handle) {
-      echo '@' . $twitter_handle;
+      $twitter_feed = get_twitter_feed($twitter_handle);
```

Comment:
would it not be better that this is inside the if? that way if there is an error and it returns false nothing happens at all